pkgs/development/python-modules/debugpy/default.nix

```diff
@@ -139,7 +139,7 @@ buildPythonPackage rec {
   meta = with lib; {
     description = "Implementation of the Debug Adapter Protocol for Python";
     homepage = "https://github.com/microsoft/debugpy";
-    changelog = "https://github.com/microsoft/debugpy/releases/tag/v${version}";
+    changelog = "https://github.com/microsoft/debugpy/releases/tag/${src.tag}";
```

Comment:
I don't think this change was required.

Reply:
Oh yep, it's not required, but this was changed by the python update script. It produces the same url, so I decided to keep it in.